packages/client/src/components/Background/Background.tsx

```diff
@@ -1,16 +1,22 @@
 import { FC, PropsWithChildren } from "react";
 import { Box } from "@mui/system";
 
-import bg from "../../assets/images/game-main-menu-bg.jpg";
+import darkBg from "../../assets/images/dark-theme/game-main-menu-bg.jpg";
+import lightBg from "../../assets/images/light-theme/game-main-menu-bg.jpg";
 
 import global from "../../global/styles/Global.module.scss";
 
-type T_BackgroundProps = PropsWithChildren<{ src?: string; isWhiteBg?: boolean }>;
+type T_BackgroundProps = PropsWithChildren<{ src?: string; isWhiteBg?: boolean; whiteBg?: string }>;
 
-export const Background: FC<T_BackgroundProps> = ({ children, src = bg, isWhiteBg = false }) => {
+export const Background: FC<T_BackgroundProps> = ({ children, src = darkBg, isWhiteBg = false, whiteBg = lightBg }) => {
+  isWhiteBg = localStorage.getItem("theme") === "light";
   return (
     <Box className={global["container"]}>
-      {!isWhiteBg ? <img src={src} alt={"background"} className={global["background"]} /> : null}
+      {!isWhiteBg ? (
+        <img src={src} alt={"background"} className={global["background"]} />
```

Comment:
Можно тернарник использовать внутри атрибута или вынести в отдельную переменную.
```js
const imgSrc = isWhiteBg ? whiteBg : src
```
```suggestion
        <img src={imgSrc} alt={"background"} className={global["background"]} />
```